procedures/LVV-T2572 - Rotator Duty Cycle Tests.ipynb

```diff
@@ -0,0 +1,822 @@
+{
```

Comment:
<p>This one is going to the same target 240 times, simply resetting the rotator angle. I think what we should probably do is move the az/el -&gt; ra/dec computation to inside the loop and maybe add a random az/el offset of about 1.5 degrees each. This would, in principle, perform a random walk around the initial az/el, which is pretty similar to what we will do on the main survey (in terms of telescope motion).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/77/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>